client/src/utils/validator.ts

```diff
@@ -2,36 +2,19 @@ import { ERRORS } from '../constants/errors';
 import { VALIDATORS_STRINGS } from '../constants/validatorStrings';
 import { VALIDATOR } from '../constants/validator';
 
-interface validatorsArray {
+interface ValidatorsArray {
```

Comment:
please think a bit more on interface name. your current interface is not an array, it's an object, so name cannot include 'array' word (exception - 'ArrayItem' or something like that)